test/SmokeTests/OOProcSmokeTests/DotNetIsolated/GreetUser.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace DotNetIsolated;
+
+public static class GreetUser
+{
+    [Function(nameof(GreetUser))]
+    public static async Task<HttpResponseData> Start(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "post")] HttpRequestData req,
+        [DurableClient] DurableClientContext durableContext,
+        FunctionContext executionContext)
+    {
+        ILogger logger = executionContext.GetLogger(nameof(GreetUser));
+
+        SayHelloPayload? payload = await req.ReadFromJsonAsync<SayHelloPayload>();
+        string instanceId = await durableContext.Client
+            .ScheduleNewOrchestrationInstanceAsync(nameof(GreetUserOrchestration), payload);
+        logger.LogInformation("Created new orchestration with instance ID = {instanceId}", instanceId);
+
+        return durableContext.CreateCheckStatusResponse(req, instanceId);
+    }
+
+    [Function(nameof(GreetUserOrchestration))]
+    public static async Task<SayHelloPayload> GreetUserOrchestration([OrchestrationTrigger] TaskOrchestrationContext context)
+    {
+        SayHelloPayload? input = context.GetInput<SayHelloPayload>();
```

Comment:
I think we need to start looking into how to write unit tests for .NET Isolated DF sometime soon.

Reply:
Agreed.